src/Chapter3.hs

```diff
@@ -479,6 +507,41 @@ After defining the city, implement the following functions:
    complicated task, walls can be built only if the city has a castle
    and at least 10 living __people__ inside in all houses of the city totally.
 -}
+data peopleInHouse = One | Two | Three | Four
+countPeople :: peopleInHouse -> Int
```

Comment:
I see that this code doesn't compile :thinking: Data type names should start with an uppercase letter according to Haskell naming rules
```suggestion
data PeopleInHouse = One | Two | Three | Four
countPeople :: PeopleInHouse -> Int
```